src/SelfServe/SelfServeUtils.tsx

```diff
@@ -0,0 +1,191 @@
+import "reflect-metadata";
+import {
+  DropdownItem,
+  Node,
+  Info,
+  InputTypeValue,
+  Descriptor,
+  AnyInput,
+  NumberInput,
+  StringInput,
+  BooleanInput,
+  DropdownInput,
+  InputType
+} from "../Explorer/Controls/SmartUi/SmartUiComponent";
+
+export enum SelfServeTypes {
+  none = "none",
```

Comment:
What is the "none" type doing? Best I can tell it rendering nothing. Seems like a job for a boolean? "showSelfServe" or something like that. Maybe I am not understanding its purpose.